notebooks/4.visualization/nbconverted/4.visualization.r

```diff
@@ -242,20 +242,20 @@ final_model_cm <- cm_df %>%
 
 final_model_cm$dataset_type <- factor(final_model_cm$dataset_type, levels = c("Train", "Test", "Plate holdout", "Well holdout"))
 
-# Define the desired order of x-axis labels
+# # Define the desired order of x-axis labels
```

Comment:
looks like these comments are unnecessary?

Reply:
(I mean the extra `#`)

Reply:
removed! thanks!